homeassistant/components/weatherflow/sensor.py

```diff
@@ -0,0 +1,417 @@
+"""Sensors for the smartweatherudp integration."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass, field
+from datetime import datetime
+from enum import Enum
+
+from pyweatherflowudp.calc import Quantity
+from pyweatherflowudp.const import EVENT_RAPID_WIND
+from pyweatherflowudp.device import (
+    EVENT_OBSERVATION,
+    EVENT_STATUS_UPDATE,
+    WeatherFlowDevice,
+)
+
+from homeassistant.components.sensor import (
+    DOMAIN as SENSOR_DOMAIN,
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    DEGREE,
+    LIGHT_LUX,
+    PERCENTAGE,
+    SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+    UV_INDEX,
+    EntityCategory,
+    UnitOfElectricPotential,
+    UnitOfIrradiance,
+    UnitOfLength,
+    UnitOfPrecipitationDepth,
+    UnitOfPressure,
+    UnitOfSpeed,
+    UnitOfTemperature,
+    UnitOfVolumetricFlux,
+)
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+from homeassistant.util.unit_system import METRIC_SYSTEM
+
+from .const import DOMAIN, LOGGER
+
+
+@dataclass
+class WeatherFlowSensorEntityDescription(SensorEntityDescription):
+    """Describes a WeatherFlow sensor entity description."""
+
+    event_subscriptions: list[str] = field(default_factory=lambda: [EVENT_OBSERVATION])
+    imperial_suggested_unit: None | str = None
+
+    backing_library_attribute_fn: Callable[[], str] = lambda: ""
```

Comment:
I don't think this is what @bdraco had in mind when referring to a lambda in https://github.com/home-assistant/core/pull/75530#discussion_r1329250224, but maybe I'm wrong.

Reply:
I'm really not sure either ... but maybe (@bdraco) can enlighten us ... i think the previous implementation was better... -> and if typing was important I could also just use `str` as the only type and `''` vs an actual item...